cypress/support/commands/parent-commands.ts

```diff
@@ -6,82 +10,6 @@
  * @see https://on.cypress.io/custom-commands#Parent-Commands
  */
 
```

Comment:
ℹ️  Moved these to a directory `@types` in the cypress dir so leverage TS's default behavior of looking for and successfully finding type declarations in `@types` folders.